packages/vitest/src/node/cli.ts

```diff
@@ -192,7 +192,7 @@ async function start(mode: VitestRunMode, cliFilters: string[], options: CliOpti
   catch {}
 
   try {
-    const ctx = await startVitest(mode, cliFilters.map(normalize), normalizeCliOptions(options))
+    const ctx = await startVitest(mode, cliFilters.map(cliFilter => normalize(cliFilter.toLocaleLowerCase())), normalizeCliOptions(options))
```

Comment:
What happens if the actual file has an uppercase letter? Would this filter work?

```
vitest SOME-name.js

./some-name.js
./SOME-name.js
```

Reply:
> What happens if the actual file has an uppercase letter? Would this filter work?
> 
> ```
> vitest SOME-name.js
> 
> ./some-name.js
> ./SOME-name.js
> ```

There were problems with uppercase letter files,  I made changes.